app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Card, CardContent } from '@material-ui/core';
+import { CardContentComponent } from './CardContext';
+import { explorationCardStyles } from './styles';
+
+const OngoingExplorationCard = ({ itemData }) => {
+  const cardData = itemData;
+  const classes = explorationCardStyles();
+
+  return (
+    <>
+      <Card className={classes.explorationCardStyles}>
+        <CardContent className={classes.cardContent}>
+          {cardData.map((eachItem) => {
```

Comment:
add proper checks for cardData
no need to use cardData, we can directly use itemData

Reply:
removed cardData.